src/pages/index.js

```diff
@@ -103,12 +163,23 @@ profileEditButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
-//add card popup
+/* ---------------------------- add card popup ---------------------------- */
 
 const handleAddCardFormSubmit = (inputValues) => {
-  const card = renderCard(inputValues);
-  section.addItem(card);
-  newCardPopup.close();
+  newCardPopup.renderLoading(true, "Saving...");
+  api
+    .addNewCard(inputValues)
+    .then(() => {
+      const card = renderCard(inputValues);
```

Comment:
Needs correction: I still can't remove or like the new added cards. You should use the data from the response to create the card, so you will have the card id